ExampleMod/Content/DamageClasses/ExampleDamageClass.cs

```diff
@@ -8,5 +9,57 @@ public class ExampleDamageClass : DamageClass
 			// Make weapons with this damage type have a tooltip of 'X example damage'.
 			ClassName.SetDefault("example damage");
```

Comment:
Maybe make this upper-case? Not trying to be nitpicky

Reply:
Vanilla is lower case iirc

Reply:
please read. the ClassName is what you see in the tooltip of weapons with that damage type